pm_kext/pm_kext.vcxproj

```diff
@@ -36,19 +36,21 @@
     <PlatformToolset>WindowsKernelModeDriver10.0</PlatformToolset>
     <ConfigurationType>Driver</ConfigurationType>
     <DriverType>KMDF</DriverType>
-    <DriverTargetPlatform>Universal</DriverTargetPlatform>
+    <DriverTargetPlatform>Desktop</DriverTargetPlatform>
```

Comment:
Target platform `Desktop` is a bit confusing. It should support all windows Desktops and server.

https://learn.microsoft.com/en-us/windows-hardware/drivers/develop/target-platforms